db/migrate/20220130151301_create_offers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOffers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :offers do |t|
+      t.text :description
+      t.string :location
+      t.float :fee
+      t.date :date
+
+      t.timestamps
+    end
+  end
```

Comment:
@AristoRap we indeed need a title of the offer

Reply:
Thanks @yauhenibankouski - Just added it!